orc8r/cloud/go/services/configurator/client_api.go

```diff
@@ -657,10 +657,10 @@ func DoesInternalEntityExist(entityType, entityKey string) (bool, error) {
 }
 
 // LoadAllEntitiesOfType fetches all entities of specified type in a network
```

Comment:
Let's also include that note about how to exhaustively read all pages here at the client api, since it's more visible for callers